unittests/test_schedulers.py

```diff
@@ -515,7 +515,7 @@ def test_cancel_with_grace(minimal_job, scheduler, local_only):
 
     t_grace = datetime.now()
     minimal_job.cancel()
-    time.sleep(0.01)
+    time.sleep(0.1)
```

Comment:
I did not understand why there is an increase in all these sleep times...

Reply:
Sometimes on loaded systems you might have occasional failures, that's why.